Philips.CodeAnalysis.Test/MsTest/TestContextAnalyzerTest.cs

```diff
@@ -20,11 +15,11 @@ namespace Philips.CodeAnalysis.Test.MsTest
 	[TestClass]
 	public class TestContextAnalyzerTest : CodeFixVerifier
 	{
-		protected override MetadataReference[] GetMetadataReferences()
+		protected override ImmutableArray<MetadataReference> GetMetadataReferences()
 		{
 			string testContextReference = typeof(TestContext).Assembly.Location;
```

Comment:
Maybe this one too idk.

Reply:
This one is actually required, kept it in.